geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreFactory.scala

```diff
@@ -35,7 +36,19 @@ class HBaseDataStoreFactory extends DataStoreFactorySpi with LazyLogging {
 
   // TODO: investigate multiple HBase connections per jvm
   private lazy val globalConnection: Connection = {
-    val ret = ConnectionFactory.createConnection(HBaseConfiguration.create())
+    val conf = HBaseConfiguration.create()
+    val auth = conf.get("hbase.security.authentication")
+    auth match{
+      case "kerberos" => {
+        conf.set("hadoop.security.authentication", "Kerberos")
+        UserGroupInformation.setConfiguration(conf)
+        UserGroupInformation.loginUserFromKeytab(conf.get("hbase.geomesa.principal"), conf.get("hbase.geomesa.keytab"))
```

Comment:
I'd like to see GeoMesa use a similar approach for Kerberos for Accumulo and HBase.  @jrs53 did some initial work on Accumulo (with docs here: http://www.geomesa.org/documentation/current/user/accumulo/kerberos.html).  

In his approach, there are two GeoTools DataStoreFactory params (username and keytab) which are passed in when the datastore is created.

My preference would be to accept these values as data store params.  I'm not opposed to being able to read them from a geomesa-site.xml for default use cases.